### DISCUSSION TITLE
How do I run this after installing python and pip3 on windows?

### DISCUSSION THREAD

**Discussion by nintendowii111** [icloud-photos-downloader/icloud_photos_downloader#478] [Q&A] [answered]
## Summary
[NOTE]: # ( Provide a brief overview of what the new feature is all about )
Sorry for noob question. See Headline above

**Comment by modjke** [comment, marked as answer]
Out of the scope of this repository, but here is a shortest way possible:
1. Uninstall python from Microsoft Store and any other python versions if any
2. Install chocolately (follow instructions on their site)
3. Stop powershell, run it as administrator (elevated) `choco install python --version=3.6.7`
4. `python --version` should be `3.6.7`
5. `pip install icloudpd`
6. Stop powershell, run from ordinary user (non-elevated) `icloudpd` will work as it should
